openedx_tagging/core/tagging/import_export/api.py

```diff
@@ -178,20 +177,20 @@ def _get_last_import_task(taxonomy: Taxonomy) -> TagImportTask | None:
     )
 
 
-def _import_export_validations(taxonomy: Taxonomy):
+def _import_validations(taxonomy: Taxonomy):
     """
     Validates if the taxonomy is allowed to import or export tags
     """
     taxonomy = taxonomy.cast()
     if taxonomy.allow_free_text:
         raise NotImplementedError(
             _(
-                "Import/export for free-form taxonomies will be implemented in the future."
+                "Import for free-form taxonomies will be implemented in the future."
```

Comment:
BTW I'm not sure we'll ever implement this... I don't even know what it would mean to import a free-form taxonomy.

Reply:
Agreed. I will change the exception in a min.